contributing/development.md

```diff
@@ -0,0 +1,99 @@
+# Development Environment Setup
+## Prerequisites
+The following applications must be installed locally to run the provider:
+- [Terraform](https://www.terraform.io/downloads.html) 1.0+ (to run acceptance tests)
+- [Go](https://golang.org/doc/install) 1.20+ (to build the provider plugin)
+
+To run the example in this repository, you will also need
+- [Docker](https://docs.docker.com/get-docker/) and [Docker Compose](https://docs.docker.com/compose/install/)
```

Comment:
I believe docker compose is included with docker now so we could remove the second part here if we want

Reply:
removed the second part of the line containing docker-compose info